CMakeLists.txt

```diff
@@ -86,6 +86,17 @@ install(TARGETS vtex2)
 
 if (BUILD_GUI)
 	install(TARGETS vtfview)
+
+	install(
+		FILES ${CMAKE_BINARY_DIR}/vtfview.desktop
+		DESTINATION share/applications
```

Comment:
is there a reason these arent absolute paths in /usr?

Reply:
These are relative to `CMAKE_INSTALL_PREFIX` and `CMAKE_INSTALL_PREFIX` might be `/usr/local`, `/usr`, `~/.local`, etc.

Reply:
cool didnt know that